requirements/quality.txt

```diff
@@ -4,11 +4,12 @@
 #
 #    pip-compile --output-file requirements/quality.txt requirements/base.in requirements/quality.in requirements/test.in
 #
+-c constraints.txt
```

Comment:
This change belongs in `requirements/quality.in` instead, and `make upgrade` needs to be run afterwards.  Note that the dual versions of astroid and pylint with different environment markers won't survive that; I don't think it's worth trying to fix pylint under Python 3.7 until after we finish moving all our repos to 3.6 and drop support for 2.7.